app/client/components/membershipFAQs.tsx

```diff
@@ -255,174 +147,33 @@ export const MembershipFAQs = (props: RouteableProps) => (
-              become a Member.
-            </p>
-
-            <p>
-              Cheques for one-off contributions towards the Guardian’s
-              journalism can be sent to one of the following addresses:
-            </p>
-
-            <p>
-              The US:<br />
-              Payable to: Guardian News & Media LLC<br />
-              Address: FAO: Finance Department, The Guardian (US Main office),
-              222 Broadway, 22nd and 23rd Floors, New York, New York, 10038
-            </p>
-
-            <p>
-              Australia:<br />
-              Payable to: GNM Australia Pty Ltd<br />
-              Address: FAO Finance Department at Guardian Australia, Level 2, 19
-              Foster Street, Surry Hills, NSW 2010
-            </p>
-
-            <p>
-              UK and Rest of World:<br />
-              Payable to: Guardian News & Media<br />
-              Address: FAO Cash Management, Finance Department. Address: Kings
-              Place, 90 York Way, London, N1 9GU.
-            </p>
-          </div>
-        </div>
-        <div title="Can I get involved in Guardian journalism?">
-          <div>
-            Details on contributing to the Guardian can be found{" "}
-            <a
-              css={{
-                textDecoration: "underline",
-                color: palette.blue.dark,
-                ":visited": { color: palette.blue.dark }
-              }}
-              href="https://www.theguardian.com/info/1999/nov/22/contributors-guide-and-contacts"
-            >
-              here
-            </a>. Readers can also get involved via{" "}
-            <a
-              css={{
-                textDecoration: "underline",
-                color: palette.blue.dark,
-                ":visited": { color: palette.blue.dark }
-              }}
-              href="https://witness.theguardian.com/"
-            >
-              GuardianWitness
-            </a>. And we regularly ask Members for their views and opinions on
-            our journalism and how we cover ongoing stories and issues.
-          </div>
-        </div>
-        <div title="I can't afford to pay you anything yet, but would like to offer my support.">
-          <div>
-            You can join Guardian Membership as a friend for free to receive
-            regular updates.
-          </div>
-        </div>
-        <div title="How does membership fit with subscriptions?">
-          <div>
-            We are currently reviewing how membership is priced, what it
-            includes/involves, and how it fits with the various print and
-            digital content packages that we offer readers, including
-            subscriptions.
-          </div>
-        </div>
-        <div title="I am under 18 - can I still be a Member?">
-          <div>No. You have to be 18 or over to join.</div>
+          <div>Yes, we now accept payments via Paypal.</div>
```

Comment:
Wow, these really haven't been updated in a while! 😅 

https://github.com/guardian/membership-frontend/pull/1519